crates/stackable-telemetry/CHANGELOG.md

```diff
@@ -30,10 +30,12 @@ All notable changes to this project will be documented in this file.
   - `FileLogSettingsBuilder::with_max_log_files` which took a `usize` was renamed to
     `FileLogSettingsBuilder::with_max_files` and now takes an `impl Into<Option<usize>>`
     for improved builder ergonomics.
+- Bump `opentelemetry` and related crates to `0.29.x` and `tracing-opentelemetry` to `0.30.0` ([#1021]).
```

Comment:
Nice, thank you! However, this needs to be moved into the "Unreleased" section instead.

Reply:
Good catch, sorry!